docs/content/routing/entrypoints.md

```diff
@@ -737,3 +737,59 @@ entrypoints.websecure.http.tls.domains[1].sans=foo.test.com,bar.test.com
     entrypoints.websecure.address=:443
     entrypoints.websecure.http.tls.certResolver=leresolver
     ```
+
```

Comment:
This should be moved to line 197 where the binding configuration is.

Reply:
True. I did not see the new chapter `address`, it definitevely belongs there.

Reply:
moved